imod/mf6/simulation.py

```diff
@@ -989,15 +990,19 @@ def _add_modelsplit_exchanges(self, exchanges_list: List[GWFGWF]) -> None:
     def _set_exchange_options(self):
         # collect some options that we will auto-set
         for exchange in self["split_exchanges"]:
-            model_name_1 = exchange.dataset["model_name_1"].values[()]
-            model_1 = self[model_name_1]
-            exchange.set_options(
-                save_flows=model_1["oc"].is_budget_output,
-                dewatered=model_1["npf"].is_dewatered,
-                variablecv=model_1["npf"].is_variable_vertical_conductance,
-                xt3d=model_1["npf"].get_xt3d_option(),
-                newton=model_1.is_use_newton(),
-            )
+            if isinstance(exchange, GWFGWF):
+                model_name_1 = exchange.dataset["model_name_1"].values[()]
+                model_1 = self[model_name_1]
+            if isinstance(exchange, GWFGWT):
```

Comment:
You don't need to add this if statement do you?

Reply:
see Joeri's remark. This if-statement is not necessary in the current setup, but if we would later use this  function at another stage in the program execution, it would break. It raises questions if it's not there.